### DISCUSSION TITLE
Preparing the v7.1.0 release

### DISCUSSION THREAD

**Discussion by vmiklos** [vmiklos/plees-tracker#101] [General]
Hi,

I plan to release v7.1.0 towards the end of the next week. It currently contains some English-only strings.

@sguinetti @fabianski7 may I ask you to update the translations you already created in the past?

Test APK here: https://github.com/vmiklos/plees-tracker/releases/tag/v7.1.0-rc2

Thanks,

Miklos

**Comment by sguinetti** [comment]
Hello @vmiklos . I will see if I can update some strings in the new version.

**Comment by fabianski7** [comment]
Thank you for notifying. Done
